scripts/rpm_owners/set_github_code_owners.py

```diff
@@ -7,13 +7,19 @@
 import re
 import sys
 from contextlib import suppress
+from subprocess import check_output
 
 import github as gh
 from github.Repository import Repository
 
 BRANCHES = ['master']
 CODEOWNERS = '.github/CODEOWNERS'
+USER = check_output(['git', 'config', 'user.name']).decode().strip()
+EMAIL = check_output(['git', 'config', 'user.email']).decode().strip()
```

Comment:
Maybe assert that USER and EMAIL are juste one line of text, just in case we get weird git output?